xml/dm_registers.xml

```diff
@@ -636,7 +661,7 @@
             \item ``Return'' the data.
             \item Mark the register stale.
```

Comment:
there is a lot of abstract discussion about the register being marked stale, but as a concept it's never really introduced (I think). It's sort of confusing because it's not like there is a "stale" bit in a register that can actually be read.

Reply:
Thinking about this more, I don't think we need the stale bit at all. Instead we can just look at the busy bit instead.